php_simdjson.cpp

```diff
@@ -220,6 +220,8 @@ PHP_MINFO_FUNCTION (simdjson) {
 
     php_info_print_table_row(2, "Version", PHP_SIMDJSON_VERSION);
     php_info_print_table_row(2, "Support", SIMDJSON_SUPPORT_URL);
+    php_info_print_table_row(2, "Implementation", simdjson::active_implementation->description().c_str());
```

Comment:
This added line breaks the complication from source (Ubuntu 22.04.5 / GCC 11.2.0):
php_simdjson.cpp:250:61: error: ‘active_implementation’ is not a member of ‘simdjson’; did you mean ‘get_active_implementation’?
250 |     php_info_print_table_row(2, "Implementation", simdjson::active_implementation->description().c_str());